Doc/library/unittest.mock.rst

```diff
@@ -303,14 +303,14 @@ the *new_callable* argument to :func:`patch`.
 
     .. method:: assert_called_once_with(*args, **kwargs)
 
-       Assert that the mock was called exactly once and with the specified
-       arguments.
+       Assert that the mock was called exactly once and that that call was
```

Comment:
I understand that it does look a bit weird at first, but it is acceptable English: http://english.stackexchange.com/questions/3418/how-do-you-handle-that-that-the-double-that-problem

It could be rephrased, but that would make it longer/more complex.